ios/RN/RNFaceDetectorModuleMLKit.m

```diff
@@ -119,9 +119,25 @@ - (int)exifOrientationFor:(UIImageOrientation)orientation
 @end
 #else
 @implementation RNFaceDetectorModuleMLKit
+
+@synthesize bridge = _bridge;
+
+- (void)setBridge:(RCTBridge *)bridge
+{
+    _bridge = bridge;
+}
+
++ (BOOL)requiresMainQueueSetup
+{
+    return NO;
+}
+
 - (NSDictionary *)constantsToExport
 {
-    return [FaceDetectorManagerMlkit constants];
+    return @{};
```

Comment:
is this correct?

Reply:
Well - the whole file depends on FirebaseMLVision, so that's why the `#if __has_include(<FirebaseMLVision/FirebaseMLVision.h>)` is in there. I'm not sure why the `#else` part with a dummy module is in there and I can't find any references to `RNFaceDetectorModuleMLKit` in the javascript side.

This right now does not change behaviour but fixes warnings.

As a clean solution I would rather put the #if completely around the module, so that `NativeModules.RNFaceDetectorModuleMLKit` does not exist if FirebaseMLVision is not linked.

Reply:
I think this is good enough for now